photon-ml/src/test/scala/com/linkedin/photon/ml/algorithm/CoordinateDescentTest.scala

```diff
@@ -62,23 +59,24 @@ class CoordinateDescentTest {
     }
 
     // KeyValueScore mock setup
-    doReturn(score).when(score).+(score)
+    doReturn(score).when(score). +(score)
     doReturn(score).when(score).setName(Matchers.any(classOf[String]))
     doReturn(score).when(score).persistRDD(Matchers.any())
 
     // Per-coordinate mock setup
     (coordinates zip models).map { case ((coordinateId, coordinate), model) =>
-      // GAMEModel mock setup
-      doReturn(gameModel).when(gameModel).updateModel(coordinateId, model)
-      doReturn(Some(model)).when(gameModel).getModel(coordinateId)
 
       // Coordinate mock setup
       doReturn(score).when(coordinate).score(model)
       doReturn((model, tracker)).when(coordinate).updateModel(model)
       doReturn((model, tracker)).when(coordinate).updateModel(model, score)
+
+      // GAMEModel mock setup
+      doReturn(gameModel).when(gameModel).updateModel(coordinateId, model)
+      doReturn(Some(model)).when(gameModel).getModel(coordinateId)
     }
 
-    // Run coordinate descent
+    // Run coordinate descent - None = no validation
```

Comment:
Scala has a built in way to include this info without comments. You can explicitly name which parameter you're setting:
`new CoordinateDescent(..., validatingDataAndEvaluatorsOption = None, ...)`
It's recommended when calling methods/initializing classes with hard values.